File: src/main/webapp/angular/user/services/kebudayaan.service.js
angular.module('penulisan')
        .service('KebudayaanService', function ($http) {
            return {
                getKebudayaans: function () {
                    return $http.get('/api/kebudayaan');
                },
                getKebudayaan: function (id) {
                    return $http.get('/api/kebudayaan/' + id);
                }
            }
        });

File: src/main/java/com/penulisan/ilmiah/service/KebudayaanService.java
package com.penulisan.ilmiah.service;

import com.penulisan.ilmiah.domain.Kebudayaan;

import java.util.List;

/**
 * @author Irman Juliansyah
 */
public interface KebudayaanService {

    void save(Kebudayaan kebudayaan);

    void update(Kebudayaan kebudayaan);

    void delete(Kebudayaan kebudayaan);

    Kebudayaan getKebudayaan(String idKebudayaan);

    List<Kebudayaan> getKebudayaans();

}
